lib/routes.js

```diff
@@ -46,8 +46,9 @@ const setupRoutes = function setupRoutes() {
         socket.lastHash = hash;
 
         const { errors = [], warnings = [] } = stats.toJson(statsOptions);
+        const hasErrors = Boolean(errors.length);
```

Comment:
Yeah, this doesn't change much using `Boolean`. It's more stricter perhaps ^o^
I simply reuse value

Instead of this:
```js
if (errors.length || warnings.length) {
  ...
  if (errors.length) return;
}
```
I written:
```js
const hasErrors = Boolean(errors.length);
if (hasErrors  || warnings.length) {
  ...
  if (hasErrors) return;
}
```